node/sc/sub_bridge_handler.go

```diff
@@ -307,7 +313,7 @@ func (sbh *SubBridgeHandler) broadcastServiceChainTx() {
 }
 
 // writeServiceChainTxReceipt writes the received receipts of service chain transactions.
```

Comment:
```suggestion
// writeChildChainTxReceipts writes the received receipts of service chain transactions.
```

Reply:
In this case, I think service chain is right. :)
This tx receipt is from parent chain. but the transactions is service chain transaction for anchoring child chain block data.

Reply:
Umm... isn't this description for below function?

Reply:
How about change the name?
`writeAnchroingTxReceipts`

Reply:
Rollback to writeServiceChainTxReceipt.